src/wallet/rpcwallet.cpp

```diff
@@ -3521,16 +3521,39 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
     // If wallet private keys are enabled, return the new transaction id,
     // otherwise return the base64-encoded unsigned PSBT of the new transaction.
```

Comment:
Do the merged changes in #22021 need to be updated with this pull?

Reply:
I'll do a rebase and also address the nits above.

Reply:
For now these comments are correct. In a multisig setup `bumpfee` will throw ` "Transaction incomplete. Try psbtbumpfee instead."`. We can improve that later.